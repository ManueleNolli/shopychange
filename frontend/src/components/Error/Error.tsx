import React from 'react'
import { Center, Text } from '@chakra-ui/react'

import { IoIosWarning } from 'react-icons/io'

export default function Error() {
  return (
    <Center mt="5vh">
      <IoIosWarning size={'4em'} color={'red'} />
      <Text color={'red'} as="b">
        Error! Please try again
      </Text>
    </Center>
  )
}
