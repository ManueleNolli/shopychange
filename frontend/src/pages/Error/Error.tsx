import React from 'react'
import { Center } from '@chakra-ui/react'
import Error from '../../components/Error/Error'

export default function ErrorPage() {
  return (
    <Center>
      <Error />
    </Center>
  )
}
