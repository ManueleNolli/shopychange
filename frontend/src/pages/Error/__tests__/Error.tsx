import React from 'react'
import { render } from '@testing-library/react'
import Error from '../Error'

test('renders error page correctly', () => {
  const component = render(<Error />)
  expect(component).toMatchSnapshot()
})
