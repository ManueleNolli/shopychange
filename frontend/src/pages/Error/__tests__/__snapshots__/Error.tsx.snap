// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders error page correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="css-gmuwbf"
      >
        <div
          class="css-1ffdipv"
        >
          <svg
            color="red"
            fill="currentColor"
            height="4em"
            stroke="currentColor"
            stroke-width="0"
            style="color: red;"
            viewBox="0 0 512 512"
            width="4em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M228.9 79.9L51.8 403.1C40.6 423.3 55.5 448 78.9 448h354.3c23.3 0 38.2-24.7 27.1-44.9L283.1 79.9c-11.7-21.2-42.5-21.2-54.2 0zM273.6 214L270 336h-28l-3.6-122h35.2zM256 402.4c-10.7 0-19.1-8.1-19.1-18.4s8.4-18.4 19.1-18.4 19.1 8.1 19.1 18.4-8.4 18.4-19.1 18.4z"
            />
          </svg>
          <b
            class="chakra-text css-tokvmb"
          >
            Error! Please try again
          </b>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-gmuwbf"
    >
      <div
        class="css-1ffdipv"
      >
        <svg
          color="red"
          fill="currentColor"
          height="4em"
          stroke="currentColor"
          stroke-width="0"
          style="color: red;"
          viewBox="0 0 512 512"
          width="4em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M228.9 79.9L51.8 403.1C40.6 423.3 55.5 448 78.9 448h354.3c23.3 0 38.2-24.7 27.1-44.9L283.1 79.9c-11.7-21.2-42.5-21.2-54.2 0zM273.6 214L270 336h-28l-3.6-122h35.2zM256 402.4c-10.7 0-19.1-8.1-19.1-18.4s8.4-18.4 19.1-18.4 19.1 8.1 19.1 18.4-8.4 18.4-19.1 18.4z"
          />
        </svg>
        <b
          class="chakra-text css-tokvmb"
        >
          Error! Please try again
        </b>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
